src/commons/EyesUtil.ts

```diff
@@ -104,4 +105,16 @@ export class EyesUtil {
 
     return this;
   }
+
+  /**
+   * Set Debug mode
+   * Set to true the eyes.log file will appear under the root folder.
```

Comment:
update jdoc to meet the function's implementation

Reply:
done